src/test/scala/com/bazaarvoice/sswf/TestSteps.java

```diff
@@ -1,15 +1,16 @@
 package com.bazaarvoice.sswf;
 
 public enum TestSteps implements WorkflowStep {
-    INPROGRESS_STEP(1, 120, 120),
-    ;
+    INPROGRESS_STEP(120, 120) {
+        @Override public InProgressTimerFunction inProgressTimerSecondsFn() {
+            return (invocationNum, cumulativeStepDurationSeconds) -> Math.min(10 * invocationNum, 100);
```

Comment:
Might as well just maintain the semantics of the test and use a constant function that returns 1.